### DISCUSSION TITLE
[Issue] The cashier can view the admin orders

### DISCUSSION THREAD

**Discussion by Moaaz-Briek** [Blair2004/NexoPOS#1596] [Ideas]
**Describe the bug**
The cashier can view the admin orders, there are no restriction.

https://github.com/Blair2004/NexoPOS/assets/53799123/949bb682-e5e0-4d81-b283-9c73e112aa39

**To Reproduce**
1. Sign in with admin role, then place an order.
2. Go to orders to see the orders list.
3. Logout.
4. Sign in as cashier, Go to Orders => Orders List, You will see the last order assigned by admin

**Expected behavior**
The cashier should only see the orders created by himself, and The admin can view all orders.

**Another Problem**
When we go to POS and click on Customers Tab, You will see all orders created by anyone for that customer, which is another problem, because the cashier can also view admin orders for that customer.

https://github.com/Blair2004/NexoPOS/assets/53799123/70ea270f-adf3-43ba-8756-e6a3bc261226

**Environment:**
 - NexoPOS 4.8.17

**Comment by Blair2004** [comment]
Hi,
Thank you. Indeed, this should be a problem.

**Comment by Blair2004** [comment]
This must be added as a permission on v5

**Comment by Moaaz-Briek** [comment]
Hello Blair
Great to hear that
